en/rocky/8/admin_guide/01-presentation.md

```diff
@@ -0,0 +1,299 @@
+# Introduction to the Linux operating system
+
+In this chapter you will learn about GNU/Linux distributions.
+
+****
+**Objectives** : In this chapter you will learn how to:
+
+:heavy_check_mark: Describe the features and possible architectures of an operating system;\
+:heavy_check_mark: Recount the history of UNIX and GNU/Linux;\
+:heavy_check_mark: Choose the right Linux distribution for your needs;\
+:heavy_check_mark: Explain the philosophy of free and opensource software;\
+:heavy_check_mark: Discover the usefulness of the SHELL.\
+
+
+To explain your goals, you should only use action verbs. Because goals need to be quantifiable, measurable and verifiable.
+At the end of the document, you can add questions directly related to these objectives to validate the understanding of the subject.
+
+:checkered_flag: **generalities**, **linux**, **distributions**
+
+**Knowledge**: :star: \
+**Complexity**: :star: \
+**Reading time**: 10 minutes
+****
+
+## What is an operating system?
+
+Linux is an **operating system**.
+
```

Comment:
I think when you are defining what an operating system is, you should include more than one example. Consider: "Linux, UNIX, BSD, Windows, and MacOS are all **operating systems".

Reply:
```suggestion
## What is an operating system?

Linux, UNIX, BSD, Windows, and MacOS are all **operating systems**.
```

Reply:
Ok I changed it.